linux/xdg/build.gradle.kts

```
@@ -0,0 +1,17 @@
+plugins {
+    java
+    kotlin("jvm")
+    kotlin("kapt")
```

Please use ksp instead of kapt.
```suggestion
    id("com.google.devtools.ksp")
```